bitbucket-slack-server-integration-plugin/pom.xml

```diff
@@ -89,7 +89,13 @@
             <groupId>com.github.seratch</groupId>
             <artifactId>jslack</artifactId>
             <scope>compile</scope>
-            <version>${jslack.version}</version>
+            <version>1.5.6</version>
+        </dependency>
+        <!-- Override vulnerable gson 2.8.5 used by jslack -->
+        <dependency>
```

Comment:
If we are not using gson then we cannot be exclude this dependency from jslack

Reply:
If `gson` is excluded, it most probably be imported from the product. Version of that `gson` provided by product is unknown and it isn't guaranteed to use compatible API with the `gson` version, that `jslack` requires. So, the app behaviour would become unpredicatable. In some cases it may work fine, in other - fail in runtime.

When `gson` version is statically specified in plugin itself, we may check in tests if it works in all expected cases. It gives the confidence about the plugin reliability that is absent in case of excluding `gson` completely.